io.github.ad_on_is.Resticity.yml

```diff
@@ -0,0 +1,76 @@
+id: io.github.ad_on_is.Resticity
+runtime: org.gnome.Platform
+runtime-version: '3.38'
+sdk: org.gnome.Sdk
+
+
+command: resticity
+finish-args:
+  - --share=ipc # Share IPC namespace with the host (necessary for X11).
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri # OpenGL rendering support.
+  - --talk-name=org.freedesktop.Notifications
+  - --filesystem=home
+  - --filesystem=host
```

Comment:
as said before these filesystem permisions are:
1. broad
2. redundant.

Reply:
I understand, but in 99% of the use cases, people want to backup their home folders to an external disk, that is mounted on /run/media/<user>/Diskname, etc. 
And I think, for newcomers it might not be obvious at first, that they need to set the correct permissions.

Reply:
> what about the word "redundant" that is unclear?

So `- --filesystem=host` exposes also the users home? Didn't know that!